packages/acs-ui-common/src/telemetry.ts

```diff
@@ -31,7 +58,10 @@ export const sanitize = (version: string): string => {
  *
  * @internal
  */
-export const _getApplicationId = (): string => {
+export const _getApplicationId = (telemetryImplementationHint?: _TelemetryImplementationHint): string => {
```

Comment:
can we make the param here required:
```suggestion
export const _getApplicationId = (telemetryImplementationHint: _TelemetryImplementationHint): string => {
```

Same as `getTelemetryImplementationHint`, ideally we should always be passing a hint into them